setup.py

```diff
@@ -93,7 +93,7 @@
     python_requires=">=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*",
     install_requires=[
         'bcrypt>=3.1.3',
-        'cryptography>=1.5',
+        'cryptography>=2.5',
```

Comment:
I was hoping to make this optional - there's already an `is_supported()` function, so it would be nice if it could return false for `cryptography<2.5` as well. Could be easy, like:

```python
try:
    from cryptography.hazmat.primitives.asymmetric import x25519
except ImportError:
    x25519 = None
```

Reply:
Sounds good, I'll go ahead and add this.